### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by PivovarA**
* Add frameworks list
* Add algorithms list
* Add algorithms parameters
* Add description of configuration file

### CODE REVIEW

**Comment by teabagk7** [comment]
Great work!👍

File: xgboost/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+## How to create conda environment for benchmarking
+`conda create -n skl_bench -c conda-forge python=3.7 xgboost pandas`
```
Comment (outoftardis) [inline xgboost/README.md:2]:
```suggestion

    conda create -n skl_bench -c conda-forge python=3.7 xgboost pandas
```

File: xgboost/README.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+##  Algorithms parameters
+
+Benchmarks can be launched for each algorithm separately. The list of supported parameters for each algorithm provided below.
```
Comment (outoftardis) [inline xgboost/README.md:6]:
```suggestion
You can launch benchmarks for each algorithm separately. The table below lists all supported parameters for each algorithm.
```

File: sklearn/README.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+
+`conda create -n skl_bench -c intel python=3.7 scikit-learn pandas`
+
+and set the appropriate environment variable
+
+`export USE_DAAL4PY_SKLEARN=YES`
+
+##  Algorithms parameters
+
+Benchmarks can be launched for each algorithm separately. The list of supported parameters for each algorithm provided below.
```
Comment (outoftardis) [inline sklearn/README.md:18]:
```suggestion
You can launch benchmarks for each algorithm separately. The tables below list all supported parameters for each algorithm:

- [General](#general)
- [DBSCAN](#dbscan)
- [RandomForestClassifier](#randomforestclassifier)
- [RandomForestRegressor](#randomforestregressor)
- [pairwise_distances](#pairwise_distances)
- [KMeans](#kmeans)
- [KNeighborsClassifier](#kneighborsclassifier)
- [LinearRegression](#linearregression)
- [LogisticRegression](#logisticregression)
- [PCA](#pca)
- [Ridge Regression](#ridge)
- [SVC](#svc)
- [train_test_split](#train_test_split)
```

sidenote: rename the "General" section, e.g. "Common parameter" or something like this

File: daal4py/README.md [hunk 4]
```diff
@@ -0,0 +1,145 @@
+
+## How to create conda environment for benchmarking
+
+`conda create -n skl_bench -c intel python=3.7 daal4py pandas`
+
+##  Algorithms parameters
+
+Benchmarks can be launched for each algorithm separately. The list of supported parameters for each algorithm provided below.
```
Comment (outoftardis) [inline daal4py/README.md:8]:
```suggestion
You can launch benchmarks for each algorithm separately. The tables below list all supported parameters for each algorithm:

- [General](#general)
- [DBSCAN](#dbscan)
- [RandomForestClassifier](#randomforestclassifier)
- [RandomForestRegressor](#randomforestregressor)
- [pairwise_distances](#pairwise_distances)
- [KMeans](#kmeans)
- [KNeighborsClassifier](#kneighborsclassifier)
- [LinearRegression](#linearregression)
- [LogisticRegression](#logisticregression)
- [PCA](#pca)
- [Ridge Regression](#ridge)
- [SVC](#svc)
- [train_test_split](#train_test_split)
```

sidenote: rename the "General" section, e.g. "Common parameter" or something like this

File: cuml/README.md [hunk 5]
```diff
@@ -0,0 +1,138 @@
+
+## How to create conda environment for benchmarking
+`conda create -n skl_bench -c rapidsai -c conda-forge python=3.7 cuml pandas cudf`
+
+##  Algorithms parameters
+
+Benchmarks can be launched for each algorithm separately. The list of supported parameters for each algorithm provided below.
```
Comment (outoftardis) [inline cuml/README.md:7]:
```suggestion
You can launch benchmarks for each algorithm separately. The tables below list all supported parameters for each algorithm:

- [General](#general)
- [DBSCAN](#dbscan)
- [RandomForestClassifier](#randomforestclassifier)
- [RandomForestRegressor](#randomforestregressor)
- [pairwise_distances](#pairwise_distances)
- [KMeans](#kmeans)
- [KNeighborsClassifier](#kneighborsclassifier)
- [LinearRegression](#linearregression)
- [LogisticRegression](#logisticregression)
- [PCA](#pca)
- [Ridge Regression](#ridge)
- [SVC](#svc)
- [train_test_split](#train_test_split)
```

sidenote: rename the "General" section, e.g. "Common parameter" or something like this

File: cuml/README.md [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+
+#### General
+| parameter Name  | Type | default value | description |
```
Comment (outoftardis) [inline cuml/README.md:10]:
```suggestion
| Parameter Name  | Type | Default Value | Description |
```

same changes for other tables

File: configs/README.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+
+Benchmarks are configured by editing the config.json file.
+In the benchmarks settings can be configured: some algorithms parameters, datasets, a list of frameworks to use, and the usage of some environment variables.
+Below is a list and description of all fields in the configuration file.
+
+###  Root Config Object
+| Field Name  | Type | Description |
+| ----- | ---- |------------ |
+|omp_env| array[string] | for xgboost only. Specify an environment variable to set the number of omp threads |
+|common| [Common Object](#common-object)| **REQUIRED** common benchmarks setting: frameworks and input data settings |
```
Comment (outoftardis) [inline configs/README.md:11]: what about creating a separate column to mark whether or not the parameter is required?

Comment (PivovarA) [reply configs/README.md:18]: We discussed this. The problem is that we have some objects, for example `training` and `testing` objects, in which there are no required parameters. In this case, there will simply be an empty column that will look unaesthetically. To maintain overall consistency, we decided to simply move the word **Requred** to the start of the line.

**Pull request merged**